packages/browser-wallet/src/popup/shared/Form/Form.scss

```diff
@@ -0,0 +1,17 @@
+@mixin when-invalid {
+    &--invalid {
+        @content;
+    }
+}
+
+.form-input {
+    @include when-invalid {
+        .form-input__field {
+            border-color: red;
+        }
+    }
+}
+
+.form-error-message {
```

Comment:
So in the desktop wallet we use camel case, is this some special case, or are you suggesting we change to kebab case in this project?

Reply:
It essentially attempts to do the same as css modules, i.e. scope css rules to modules (or blocks in this case). Without css modules, it's easier to makes css rules that apply to blocks inside other blocks (override styling of nested components in a special case), as you don't have the automatic hashed scoping from css modules.